base/src/test/java/uk/ac/ebi/atlas/search/suggester/SuggesterDaoIT.java

```diff
@@ -91,7 +91,7 @@ void closestMatchIsFirst() {
         String twoCharSymbol = "AR";
 
         // Suggestion::equals is established only by term and payload, weight isn’t considered
-        assertThat(subject.fetchBioentityProperties(twoCharSymbol.toLowerCase(), 10, false))
+        assertThat(subject.fetchBioentityProperties(twoCharSymbol, 10, false))
                 .isNotEmpty()
                 .startsWith(new Suggestion(twoCharSymbol, 0, "symbol"));
         assertThat(subject.fetchBioentityProperties("ar", 10, false, speciesUtils.getHuman()))
```

Comment:
Should this be `twoCharSymbol.toLowerCase()`?

Reply:
I thought so at firsth, but later I realised that the suggestions for e.g. “asp” are different than “ASP” or “Asp”.